src/python/views/demo_bio_01_funcoup/__init__.py

```diff
@@ -0,0 +1,307 @@
+import graphistry, os, pandas as pd, streamlit as st
+from components import GraphistrySt, URLParam
+from graphistry import PyGraphistry
+from css import all_css
+from time import sleep
+import logging
+
+############################################
+#
+#   DASHBOARD SETTINGS
+#
+############################################
+#  Controls how entrypoint.py picks it up
+
+
+app_id = 'app_00'
+logger = logging.getLogger(app_id)
+urlParams = URLParam(app_id)
+
+
+def info():
+    return {
+        'id': app_id,
+        'name': 'Bio: FUNCOUP',
+        'enabled': True,
+        'tags': ['bio', 'large', 'funcoup','demo'],
+    }
+
+
+def run():
+    run_all()
+
+
+############################################
+#
+#   CUSTOM CSS
+#
+############################################
+# Have fun!
+
+def custom_css():
+
+    all_css()  # our favorites
+
+
+############################################
+#
+#   SIDEBAR RENDER AERA
+#
+############################################
+# Given URL params, render left sidebar form and return combined filter settings
+
+# #https://docs.streamlit.io/en/stable/api.html#display-interactive-widgets
+def sidebar_area():
```

Comment:
A bit more description in the side bar and main area on what is being shown may help... what/why wrt topic + how it is rendered, as may not be obvious to everyone